infrastructure/templates/infrastructure/powerplant_detail.html

```diff
@@ -121,9 +121,20 @@ <h3>Estimated Output</h3>
       <div class="row double">
           <section>
               <h3>Fuel</h3>
-              {% for fuel in object.fuels.all %}
-                <div>{{ fuel }}</div>
-              {% endfor %}
+              <div class="row double">
+                <section>
+                    <h4> Fuel Type </h4>
+                    {% for fuel in object.fuels.all %}
+                        <div> {{ fuel }}</div>
```

Comment:
I was thinking we should stick pretty close the existing implementation and just add category to the output, e.g. something like ``<div>{{ fuel }} ({{ fuel.fuel_category }})</div>`` and remove the 2nd loop here.

Reply:
My main reason is that with two information pieces it is better to be explicit about what you are showing someone. I'm fine moving it back to the single loop version.

Reply:
Maybe to remove the second loop do, but keep explicit typing do something like ?

```<h3> Fuels </h3>
<span style="padding,font">Fuel Type:</span> {{ fuel }} <span style="padding, font">Fuel Category: </span> {{ fuel.fuel_category }}

```

Reply:
I believe this has been resolved by Maesae's comments in basecamp.